go.mod

```diff
@@ -11,20 +11,21 @@ require (
 	github.com/openshift/cluster-logging-operator/api/observability v0.0.0-20250228165433-3665fb550c08
 	github.com/operator-framework/api v0.26.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.76.2
+	github.com/rhobs/observability-operator v1.0.0
```

Comment:
Maybe it would be good for the COO/OBO to introduce a new API module similarly to prometheus-operator? :thinking: